internal/api/mocks/schema_service.go

```diff
@@ -32,14 +41,21 @@ func (_m *SchemaService) CheckCompatibility(ctx context.Context, nsName string,
 func (_m *SchemaService) Create(ctx context.Context, nsName string, schemaName string, metadata *schema.Metadata, data []byte) (schema.SchemaInfo, error) {
 	ret := _m.Called(ctx, nsName, schemaName, metadata, data)
 
+	if len(ret) == 0 {
+		panic("no return value specified for Create")
+	}
+
```

Comment:
can we return an error instead of panic, since this method returns an error anyways?